packages/react-integration/demo-app-ts/src/components/demos/ActionListDemo/ActionListDemo.tsx

```diff
@@ -1,104 +0,0 @@
-import {
-  ActionList,
-  ActionListGroup,
-  ActionListItem,
-  Button,
-  Dropdown,
-  DropdownItem,
-  DropdownSeparator,
-  KebabToggle
```

Comment:
Are you removing these demos because the types are being verified in the example code now since you converted to typescript?

Reply:
No, see my earlier comment here: https://github.com/patternfly/patternfly-react/pull/6345#issuecomment-926715093

We do a good job of not breaking types and these demos do a poor job of checking types because the demo app builds in CI 97% of the time over 264 runs. I think it safe to remove, especially because we will have proper example typechecking as soon as we begin conversion.

Reply:
Maybe we keep them until we do the conversion?  We can open an issue to come back so we don't loose track.

Reply:
During the conversion we'd be removing these 89 demos anyways. I think it's bad to keep demos which have no corresponding integration tests, are a copy/paste of existing demos, and offer little type safety. No type problems that we find downstream in products like Openshift are captured by these demos.

As an example, here's `TileDemo`: `<Tile title="Selected" id="selected" isSelected />`. If any of those props were to change in a PR we'd see it in the interface and in existing examples in the docs. Having this demo provides no type safety.

Reply:
Ok. I see what you are saying.  It was definitely useful before typescript conversion.  So these should be ok to remove.
Did you see my question about testing all variations of breakpoints.  If we remove here, we should cover in unit test.  Other than that we, should be good to go.

Reply:
The only breakpoint utility checks I removed are ones that only check the default viewport size. I kept the test that checks all the different viewport sizes.